lib/api.c

```diff
@@ -137,6 +137,7 @@ int request_table_ver_and_size(ryzen_access ry)
 	case FAM_LUCIENNE:
 	case FAM_CEZANNE:
 	case FAM_REMBRANDT:
+	case FAM_PHEONIX:
```

Comment:
Did you mean `PHOENIX` here and not `PHEONIX`?

Reply:
Autocompletion... I also noticed that. But the typo wasn't introduced by me.

Reply:
I will fix the typo later.